java-client/build.gradle.kts

```diff
@@ -142,8 +142,8 @@ val integrationTest = task<Test>("integrationTest") {
 dependencies {
 
     val opensearchVersion = "3.0.0-SNAPSHOT"
-    val jacksonVersion = "2.13.4"
-    val jacksonDatabindVersion = "2.13.4.2"
+    val jacksonVersion = "2.14.0"
+    val jacksonDatabindVersion = "2.14.0"
```

Comment:
@VachaShah doesn't dependabot support upgrading these files?

Quick search says it does support Kotlin[1].

[1] https://github.com/dependabot/dependabot-core/issues/2238